src/shared/services/hashing.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common'
+import { compare, hash } from 'bcrypt'
+import { SALT_ROUNDS } from '../constants/auth.constant'
+
+const saltRounds = SALT_ROUNDS
+
+@Injectable()
+export class HashingService {
+  hash(value: string) {
+    return hash(value, saltRounds)
```

Comment:
```suggestion
    return hash(value, SALT_ROUNDS)
```
mà sẽ dùng trực tiếp ở đây luôn